storio/src/androidTest/java/com/pushtorefresh/storio/db/integration_test/impl/InsertTest.java

```diff
@@ -71,4 +54,41 @@
 
         cursor.close();
     }
+
+    @Test public void insertTwice() {
```

Comment:
Test name `insertTwice()` but it's actually doing insert and delete twice, I thought it will try to do insert twice and see what will happen, may be will be better to rename test into `insertAndDeleteTwice`?